app/models/force/lottery_preference.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Force
+  # Represents a single unit object. Provide mapping between
+  # Salesforce object field names and LAP domain field names for listings.
+  class LotteryPreference < Force::ObjectBase
+    FIELD_NAME_MAPPINGS = [
+      { domain: 'id', salesforce: 'Id' },
+      { domain: 'total_submitted_apps', salesforce: 'Total_Submitted_Apps' },
+      { domain: 'order', salesforce: 'Order' },
+      { domain: 'description', salesforce: 'Description' },
+      { domain: 'available_units', salesforce: 'Available_Units' },
+      { domain: 'pdf_url', salesforce: 'PDF_URL' },
+      { domain: 'lottery_preference', salesforce: 'Lottery_Preference' },
+    ].freeze
+
+    def to_domain
+      domain_fields = super
+
+      domain_fields.lottery_preference = {
```

Comment:
what does this piece do?

Reply:
Logan added this one in the helper, so I supposed that was needed. `lottery_preference` is an object and it's used in `PreferenceAdditionalOptions.js`